subsystem/elevator.py

```diff
@@ -75,4 +84,17 @@ def is_at_position(self, height):
             height (float): height to be checked in meters
         """
         # Rounding to make sure it's not too precise (will cause err)
-        return round(self.get_position(), 2) == round(height, 2)
\ No newline at end of file
+        return round(self.get_position(), 2) == round(height, 2)
+    
+    def update_network_table(self):
+        table = ntcore.NetworkTableInstance.getDefault().getTable('elevator')
+
+        table.putNumber('elevator height', self.get_position())
```

Comment:
Generally, these are all going to be in a table that is marked Elevator, so we don't need to precede all the info with the word "elevator."